packages/circuits/lib/zk_transaction.circom

```diff
@@ -211,28 +211,28 @@ template ZkTransaction(tree_depth, n_i, n_o) {
     var range_limit = (0 - 1) >> 8;
     component inflow_eth_range[n_i];
     for(var i = 0; i < n_i; i ++) {
-        inflow_eth_range[i] = LessThan(254);
+        inflow_eth_range[i] = LessThan(252);
         inflow_eth_range[i].in[0] <== spending_note_eth[i];
         inflow_eth_range[i].in[1] <== range_limit;
         inflow_eth_range[i].out === 1;
     }
     component inflow_erc20_range[n_i];
     for(var i = 0; i < n_i; i ++) {
-        inflow_erc20_range[i] = LessThan(254);
+        inflow_erc20_range[i] = LessThan(252);
         inflow_erc20_range[i].in[0] <== spending_note_erc20[i];
         inflow_erc20_range[i].in[1] <== range_limit;
         inflow_erc20_range[i].out === 1;
     }
     component outflow_eth_range[n_o];
     for(var i = 0; i < n_o; i ++) {
-        outflow_eth_range[i] = LessThan(254);
+        outflow_eth_range[i] = LessThan(252);
```

Comment:
@kobigurk 
Is this enough to change n from 254 to 252?

Reply:
You also need to make sure that both inputs are already range limited to < 2^252.
 think here I'd just go with feeding `new_note_eth` (and others similarly) into Num2Bits of appropriate size (e.g. 245) and remove the use of LessThan.

Reply:
I understood what you're saying! Thanks!!

Reply:
It seems we can only use Num2Bits(245) instead of LessThan if we set the range_limit just as 2**245.